shiny-api/api/handlers.go

```diff
@@ -0,0 +1,222 @@
+package main
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/apiarian/migration-playground/shiny-api/common"
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+)
+
+type ThingView struct {
+	ID        string  `json:"id"`
+	Name      string  `json:"name"`
+	Foo       float64 `json:"foo"`
+	CreatedOn string  `json:"created-on"`
+	UpdatedOn string  `json:"updated-on"`
+	Version   string  `json:"version"`
+}
+
+func ViewThing(t *common.Thing) *ThingView {
+	if t == nil {
+		return nil
+	}
+
+	return &ThingView{
+		ID:        t.ID,
+		Name:      t.Name,
+		Foo:       t.Foo,
+		CreatedOn: t.CreatedOn.Format(time.RFC3339),
+		UpdatedOn: t.UpdatedOn.Format(time.RFC3339),
+		Version:   t.Version,
+	}
+}
+
+type ThingInput struct {
+	Name    string  `json:"name"`
+	Foo     float64 `json:"foo"`
+	Version string  `json:"version"`
+}
+
+func CodeOrDefault(err error, def int) int {
```

Comment:
Are the all exported?? 
if yes, then go lint/vet will complain about comments for these functions..